pkgs/tools/wayland/hyprevents/default.nix

```diff
@@ -0,0 +1,40 @@
+{ fetchgit, lib, makeWrapper, stdenv, hyprland, bash }:
+
+stdenv.mkDerivation rec {
+  pname = "hyprevents";
+  version = "1.0-20221209";
+
+  src = fetchgit {
+    url = "https://github.com/vilari-mickopf/hyprevents";
+    rev = "95426ae4dac70c6e6ed67918dbf3bdedfdf28f37";
+    sha256 = "sha256-SLt2ukCJbfkvnGLgy9x7/3BoI2OqC4VIdgM5z768EXc=";
+  };
+
+  patches = [ ./0001-move-event_loader-event_handler-make-it-executable.patch ];
```

Comment:
Would it be possible to upstream these changes?

Reply:
Sorry... this comment was hidden for some reason. I should probably try but the projects' maintainer doesn't seem active?